src/dev/code_coverage/ingest_coverage/ingest_code_coverage_readme.md

```diff
@@ -0,0 +1,25 @@
+
+# Convert Code Coverage Json Summary and Send to ES
+
+
+## How it works 
+
+It starts with this jenkins pipeline file:
+
+.ci/Jenkinsfile_coverage#L60
+
+```
+src/dev/code_coverage/shell_scripts/ingest_coverage.sh ${BUILD_NUMBER} ${env.BUILD_URL}
+```
+
+The above line depends on a number of things happenning before this will run properly.
```

Comment:
Would future us would probably benefit from knowing what these things are?

Reply:
Perhaps?

Reply:
Actually, I think I should drop that as that was a while ago.  Fixing up now